keps/sig-cloud-provider/20190125-out-of-tree-vsphere.md

```diff
@@ -67,33 +77,38 @@ This KEP is specifically tracking progress for the vSphere cloud provider.
 
 ## Proposal
 
+The vSphere Cloud Provider is implemented, tested and partially documented. The next major steps are to migrate existing users of the in-tree provider to the external provider.
+
 ### Implementation Details/Notes/Constraints [optional]
 
-TODO for SIG-VMware
+Main provider work is completed and feature parity with in-tree will be achieved with the beta version. Removing in-tree `vsphere_volume` code is underway in the same repo and will piggyback on and be tracked through the
+[In-tree Storage Migration to CSI Plugin Migration](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md)
+work. New features will need to follow the community KEP process.
 
 ### Risks and Mitigations
 
-TODO for SIG-VMware
+None known
 
 ## Design Details
 
 ### Test Plan
 
-TODO for SIG-VMware
+Third-party testing of the vSphere Cloud Provider is handled by VMware and reports to testgrid.
 
 ### Graduation Criteria
 
-TODO for SIG-VMware
+This feature is complete and will be ready for graduation with release 1.14.
```

Comment:
TBH I'm not sure what "graduation" for KEP means here. Does it mean https://github.com/kubernetes/enhancements/issues/670 is GA? cc @justaugustus might know?